torch/_meta_registrations.py

```diff
@@ -114,3 +114,32 @@ def meta_adaptive_avg_pool2d(self, output_size):
 def meta_adaptive_avg_pool3d(self, output_size):
     check(self.ndim == 4 or self.ndim == 5, f"Expected 4D or 5D tensor, but got {self.shape}")
     return self.new_empty(self.shape[:-3] + tuple(output_size))
+
+@torch.library.impl(meta_lib, "reflection_pad2d")
+def meta_reflection_pad2d(self, padding):
```

Comment:
this will work for various padding modes, so call it `meta_pad2d` to future-proof?

Reply:
oddly enough, none of the other padding modes have xfails in test_meta.py, maybe cuz they all got ported to structured